msteams-platform/concepts/build-and-test/apps-localization.md

```diff
@@ -38,7 +38,7 @@ The images that you upload with the English language are used in AppSource.
 
 ## Localize strings in your app manifest
 
-You must use the Microsoft Teams app schema `v1.5` and later to localize your app. You can do this by setting the `$schema` attribute in your manifest.json file to **https://developer.microsoft.com/en-us/json-schemas/teams/v1.5/MicrosoftTeams.schema.json** or higher and updating the `manifestVersion` property to `$schema` version (`1.5` in this case). 
+You must use the Microsoft Teams app schema `v1.5` and later to localize your app. You can do this by setting the `$schema` attribute in your manifest.json file to ``https://developer.microsoft.com/json-schemas/teams/v1.5/MicrosoftTeams.schema.json`` or higher and updating the `manifestVersion` property to `$schema` version (`1.5` in this case).
```

Comment:
@v-habibaumme , please make it a single backtick.

Reply:
Made the requested changes